src/common/PersonTile/styled.js

```diff
@@ -3,60 +3,44 @@ import profile from "../../images/profile.svg";
 
 export const Content = styled.li`
     cursor: pointer;
-    height: 100%;
     padding: 16px;
     background-color: ${({ theme }) => theme.color.white};
-    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    box-shadow: 0 4px 12px rgba(186, 199, 213, 0.5);
     border-radius: 5px;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    text-align: center;
     transition: 0.2s;
-
+    list-style: none;
+    display: grid;
+    grid-template-rows: repeat(3, min-content);
     &:hover {
         transform: scale(1.02);
         box-shadow: 0 4px 12px rgba(186, 199, 213, 0.9);
     }
+
     &:active {
         transform: scale(1.03);
     }
-
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
-        width: 136px;
-        padding: 8px;
-    }
 `;
 
 export const Img = styled.img`
+    width: 100%; 
+    aspect-ratio: 2/3;
     border-radius: 5px;
-    width: 176px;
-    height: 231px;
-    object-fit: cover;
-    object-position: 100%;
-    flex-shrink: 0;
-
     background-color: ${({ theme }) => theme.color.silver};
     background-image: url(${profile});
     background-position: center;
     background-repeat: no-repeat;
-
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
-        width: 120px;
-        height: 178px;
-    }
+    background-size: 35%;
 `;
 
-export const Name = styled.h2`
-    margin: 16px 0px 8px;
+export const Name = styled.h3`
```

Comment:
This was discussed in [one of @JamieShifter PRs](https://github.com/Marcin-Malek/movies-browser/pull/9#discussion_r910435391). The way it was resolved was not perfect, it would have been better if we left Name as h3 and change Role to h4 since there already is a h2 on site and we have 6 levels of headings. In my opinion these two components could be spans as well, it depends on how would we develop PeopleTile further.

Reply:
okey i missed this , fixed